apps/web-giddh/src/app/shared/template-froala/template-froala.component.ts

```diff
@@ -328,10 +361,11 @@ export class TemplateFroalaComponent implements OnInit {
     * @memberof TemplateFroalaComponent
     */
     public toggleBccCc(type: string): void {
-        if (type == "bcc") {
-            this.showBcc = !this.showBcc;
-        } else {
-            this.showCc = !this.showCc;
+        this.setEmailFocus(type);
+        if (type == EmailType.Bcc) {
```

Comment:
```suggestion
        if (type === EmailType.Bcc) {
```

Reply:
this.showBcc = type === EmailType.Bcc;

do this for all

Reply:
Done